### ISSUE TITLE
Values Being Overwritten When Using Two AzDO Tasks with JSON files Containing Unique Values 

### ISSUE THREAD

**Issue by MattMencel** [Azure/AppConfiguration#398]
I'm trying out the Azure App Configuration Push task in AzDO. Running into an issue... I have a CI label and I need to push both regular k/v properties, as well as Keyvault reference properties with the special content-type.

I run them in two tasks like so...

``` yaml
        steps:
          - task: AzureAppConfiguration.azure-app-configuration-task-push.custom-build-release-task.AzureAppConfigurationPush@1
            displayName: "AppConfig Push CI: nonprod-appconfig"
            inputs:
              azureSubscription: "nonprod-svccon"
              ConfigstoreName: "nonprod-appconfig"
              ConfigurationFile: CI.json
              Separator: ","
              Label: CI
          - task: AzureAppConfiguration.azure-app-configuration-task-push.custom-build-release-task.AzureAppConfigurationPush@1
            displayName: "AppConfig Push CI-keyvault: nonprod-appconfig"
            inputs:
              azureSubscription: "nonprod-svccon"
              ConfigstoreName: "nonprod-appconfig"
              ConfigurationFile: CI-keyvault.json
              Separator: ","
              Label: CI
              ContentType: "application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8"
```

The values in each file are unique.

I originally had Strict: true set, which deletes existing values before applying new ones. I realized however that the second task was overwriting the values of the first with that setting enabled.

I thought setting Strict: false or removing it altogether would be the fix, but when the pipeline runs, both tasks complete but the second task is still removing the values from the first.

**Comment by JonasDewit** [comment]
Had the same issue with the AppConfigurationPush task. Switched to az cli to solve it for the time being

**Comment by adriend-advz** [comment]
Had also this issue with the AppConfigurationPush task with a properties file.
Any workaround possible?

**Comment by Yiming-Jia** [comment]
Hello,

We identified the bug from the AppConfigurationPush task. And we just released a a new version (1.0.1) to fix the bug. Your task should automatically adapt to the new version. Please let us know if the bug is still persist in the latest version.

**Comment by adriend-advz** [comment]
Hello @Yiming-Jia!
Thanks for the information. Checked on my side, task has been updated.
However, I retried today, and every Key that was **not** in the properties file got erased.
Is it something I'm doing wrong?
Thanks!

**Comment by jimmyca15** [comment]
@adriend-advz 

Can you confirm you're not setting the `strict` parameter to true?

**Comment by adriend-advz** [comment]
Hi @jimmyca15!
Nope, set to false:
![image](https://user-images.githubusercontent.com/67128916/93821720-418be000-fc5f-11ea-83a0-db8645e4100e.png)

**Comment by Yiming-Jia** [comment]
Hello @adriend-advz ,

Sorry for the inconvenience here. We just released an version 1.0.2 to fix the bug you mentioned above. And the new release should be automatically adapt to your existing Azure Devops Pipeline. Please let us know if the problem still blocks you.

**Comment by adriend-advz** [comment]
Hi @Yiming-Jia ! Thanks a lot for the fix, works like a charm :)
